SPCTR-3013 Fix: Marketing Button - Resolved an issue where border width and paddings were not applied correctly in the editor's Tablet/Mobile view. (#3307)
brainstormforce/ultimate-addons-for-gutenberg: README.md

## Changelog ##

### x.x.x ###

> * Fix: Marketing Button - Resolved an issue where border width and paddings were not applied correctly in the editor's Tablet/Mobile view.

Reviewer: ```suggestion
* Fix: Marketing Button - Resolved an issue wherein the border-width and padding would not work as intended in the editor's tablet & mobile view.
```

Author: Done
